addons/scratchr2/experimental_dark_compatibility.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --scratchr2-dropdown-image: url(https://mxmou.github.io/customize-scratch/assets/caret.svg);
```

Comment:
Maybe you could save this SVG in the addon folder? I think `url()` functions in CSS are relative to the CSS file itself, not the page they're injected to, so something like `url(../caret.svg)` should work?

Reply:
But theme userstyles are injected as `<style>` elements, so the browser doesn't know which file they came from.

Reply:
Nice catch. In the future, I might add something like `%addon.self.dir%` to the userstyle injector.

Reply:
Moved discussion to #741